biztalk/install-and-config-guides/hardware-and-software-requirements-for-biztalk-server-2020.md

```diff
@@ -34,19 +34,21 @@ The following table lists the minimum hardware requirements for your BizTalk Ser
+|                      WinSCP                       |                                                                                                                WinSCP version 5.15.4. [Download WinSCP](http://winscp.net).                                                                                                                |                                                                                                                                                                                                                                                                                                                                                                                                                                                                               Required to use the SFTP adapter.                                                                                                                                                                                                                                                                                                                                                                                                                                                                               |
+|                 MQSeries adapter                  |                                                                                             <ul><li>IBM WebSphere MQ 8</li></ul>                                                                                              |                                                                                                                                                                                                                                                                                                                                                                                                                                                                   Optional.<br/>Required only when using IBM WebSphere MQ.                                                                                                                                                                                                                                                                                                                                                                                                                                                                    |
```

Comment:
Also IBM WebSphere MQ 9, this was there in 2016 too, we cannot drop it in 2020

Reply:
updated, was accidentally removed.